modelscope/pipelines/cv/anydoor_pipeline.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+
+import os
+from typing import Any, Dict
+
+import cv2
+import einops
+import numpy as np
+import torch
+
+from modelscope.metainfo import Pipelines
+from modelscope.models.cv.anydoor.cldm.ddim_hacked import DDIMSampler
+from modelscope.models.cv.anydoor.datasets.data_utils import (
+    box2squre, box_in_box, expand_bbox, expand_image_mask, get_bbox_from_mask,
+    pad_to_square, sobel)
+from modelscope.outputs import OutputKeys
+from modelscope.pipelines.base import Input, Pipeline
+from modelscope.pipelines.builder import PIPELINES
+from modelscope.utils.constant import Tasks
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.image_to_image_generation, module_name=Pipelines.anydoor)
+class AnydoorPipeline(Pipeline):
+
```

Comment:
add pipeline doc str, refer to https://github.com/modelscope/modelscope/blob/master/modelscope/pipelines/cv/card_detection_correction_pipeline.py#L34C1-L34C1

Reply:
done